### PR TITLE
[GH-#362] adjust readme

### LINKED ISSUE #362: clean readme

### PR DESCRIPTION
**Pull request by thepiwo**
closes #362

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,105 +43,103 @@ Start the application in interactive Elixir mode
...
+- Write all your specific functions and checks inside your new `Transaction module`
 
 All custom transactions are childs to the `DataTx` Transaction that wraps them inside.
 The DataTx strucure hold:
-- The name of your `transaction type` that should be you `Transaction Module name`;
-- The `payload` that will hold your `custom transaction structure`;
-- The `sender`;
-- The `nonce`;
-- The `fee`;
+- The name of your `transaction type` that should be you `Transaction Module name`
```
Comment (d-velev) [inline README.md]: I know this was like this even before the PR but the sentence doesn't make it clear, if this is the atom of the module then maybe we should say something along those lines

File: README.md [hunk 2]
```diff
@@ -282,20 +278,11 @@ the log can be found in the source folder under:`apps/aecore/logs`
...
+  `Oracle.respond(query_id, response, fee)`
   Responds to a query which is referenced by ID, if the response is added successfully, the oracle receives the query_fee contained in the query.
 
 **Extend a registered oracle:**
-  `Aecore.Oracle.Oracle.extend(ttl, fee)`
+  `Oracle.extend(ttl, fee)`
   Extends the TTL of an oracle with the address that matches the address of the node.
 
 All transactions have to be mined in order to take effect.
-
```
Comment (d-velev) [inline README.md]: Why remove the docker segment?

File: README.md (hunk 1 above)
Comment (thepiwo) [reply README.md]: @gspasov you have a proposal for this?

File: README.md (hunk 2 above)
Comment (thepiwo) [reply README.md]: It was duplicated from the top

File: README.md (hunk 1 above)
Comment (gspasov) [reply README.md]: Maybe: __The name of your new `tramsacton type` should be the name of the `Transaction Module name`__ ?

**Pull request merged**